.gitlab/kitchen_testing/debian.yml

```diff
@@ -78,6 +77,12 @@ kitchen_debian_install_script_agent-a6_arm64:
     - .kitchen_scenario_debian_a6_arm64
     - .kitchen_test_install_script_agent
 
+kitchen_debian_install_script_iot_agent-a7:
+  rules: !reference [.on_default_kitchen_tests_a7]
+  extends:
+    - .kitchen_scenario_debian_a7_x64
+    - .kitchen_test_install_script_iot_agent
```

Comment:
```suggestion
```
I think you do not need to add the kitchen iot agent test back, we now have the same test running on new-e2e! [Here](https://github.com/DataDog/datadog-agent/blob/main/.gitlab/new-e2e_testing/debian.yml#L48-L60)

Reply:
Resolved in [8694935](https://github.com/DataDog/datadog-agent/pull/21127/commits/8694935e94d05b45cc01362a1c359a5d8a90860b)